src/main/java/net/pms/network/RequestHandler.java

```diff
@@ -104,17 +104,8 @@ public void run() {
 			// header matches are attempted and if those fail as well we're stuck with the
 			// default renderer.
 
-			// Attempt 1: If the reguested url contains the no-transcode tag, force
-			// the default streaming-only conf.
-			if (request.getArgument().contains(RendererConfiguration.NOTRANSCODE)) {
```

Comment:
`request` is always null at this line.

Reply:
@valib cool, can you do that as part of this branch, or should I?

Reply:
@SubJunk it would be better if you do it

Reply:
That's done now, can you review?